extensions/blocks/premium-content/premium-content.php

```diff
@@ -32,6 +31,7 @@ function register_block() {
 		FEATURE_NAME,
 		array(
 			'render_callback' => __NAMESPACE__ . '\render_block',
+			'enable_preview'  => true,
```

Comment:
Taking suggestions for a better name for this option.

Reply:
`enable_frontend_preview` might be slightly more explanatory?